tests/fixtures/database.py

```diff
@@ -938,7 +938,7 @@ def temporary_directory_configuration() -> Iterable[
 
 
 @pytest.fixture(scope="session")
-def application() -> Iterable[ApplicationFixture]:
+def application(mock_config_key_pair: None) -> Iterable[ApplicationFixture]:
```

Comment:
The new fixtures implementation follows the coding style of documentation through type definition, so maybe return something in the fixture of a suitable type? Like the key pair? or an object `class Keypair` that just holds the keypair data?

Reply:
I'm not sure I 100% understand the convention we are using for fixtures. 

I rewrote this fixture to return the key pair as part of a dataclass. @RishiDiwanTT @io7m can you take another look and let me know if this looks better?